aws/vpc.tf

```diff
@@ -39,3 +39,17 @@ data aws_subnet public {
    id       = each.value
```

Comment:
did you intend to check vpc.tf in? isn't it just a troubleshooting file?

Reply:
There was a "vpc" troubleshooting *directory* which had the job of evaluating the name lookup magic fast to generate outputs so the results of name configuration could be verified  without attempting to deploy an entire cluster and  waiting for an hour.   vpc.tf is where the magic for "importing" the vpc and  subnets  using "data" constructs is located. 
 I think most likely we're going to settle in on standard naming patterns and I can  imagine using the same template instantiation for DEV,TEST,OPS.  Maybe even all projects.